lib/Component/VEvent.php

```diff
@@ -60,6 +63,7 @@ public function isInTimeRange(\DateTime $start, \DateTime $end) {
         } elseif (!$this->DTSTART->hasTime()) {
             $effectiveEnd = clone $effectiveStart;
             $effectiveEnd->modify('+1 day');
+            $effectiveEnd->modify('-1 second');
```

Comment:
Could we use a modifier like `last second of this day` or `front of tomorrow`? Not tested but interesting though.